File: app/src/main/java/com/bonustrack02/numberbaseballgametp/BeginningActivity.java
package com.bonustrack02.numberbaseballgametp;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.bonustrack02.numberbaseballgametp.databinding.ActivityBeginningBinding;

public class BeginningActivity extends AppCompatActivity {

    ActivityBeginningBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityBeginningBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: app/src/main/java/com/bonustrack02/numberbaseballgametp/BeginningActivity.kt
package com.bonustrack02.numberbaseballgametp

import android.content.Intent
import android.os.Bundle
import android.os.Handler
import android.os.Looper
import android.os.Message
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.bonustrack02.numberbaseballgametp.databinding.ActivityBeginningBinding

class BeginningActivity : AppCompatActivity() {
    val binding: ActivityBeginningBinding by lazy { ActivityBeginningBinding.inflate(layoutInflater) }
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_FULLSCREEN
        handler.sendEmptyMessageDelayed(0, 2500)
    }

    val handler: Handler = object : Handler(Looper.getMainLooper()) {
        override fun handleMessage(msg: Message) {
            val intent = Intent(this@BeginningActivity, MainActivity::class.java)
            startActivity(intent)
            finish()
        }
    }
}